.travis/run_tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+if [[ $TRAVIS ]]; then
+    cd "$TRAVIS_BUILD_DIR" || exit 2
+
+    # Check modified PHP files with PHP's internal syntax checker
+    git diff --name-only HEAD^ | grep '\.php$' | xargs -r -n 1 php -l || exit 1
```

Comment:
So this runs on the _whole_ of any file that has been changed, right?

Reply:
Yup.

Reply:
Is that for efficiency, or are there some files that would currently fail?

Reply:
Not for efficiency, but it seems cruel to report errors on files that have not been changed.

Reply:
:sparkles: Fairy Nuff. :sparkles: